administration/installation-and-deployment/post-install-actions/uninstall-system-install.md

```diff
@@ -14,9 +14,9 @@ NSO can be uninstalled using the [ncs-installer(1)](https://developer.cisco.com/
 # ncs-uninstall --all
 ```
 
-Executing the above command removes the Installation Directory `/opt/ncs` including symbolic links, Configuration Directory `/etc/ncs`, Run Directory `/var/opt/ncs`, Log Directory `/var/log/ncs`, init scripts from `/etc/init.d` and user profile scripts from `/etc/profile.d`.
+Executing the above command removes the Installation Directory `/opt/ncs` including symbolic links, Configuration Directory `/etc/ncs`, Run Directory `/var/opt/ncs`, Log Directory `/var/log/ncs`, `systemd` service file `/etc/systemd/system/ncs.service`, `systemd` package reload environment file `/etc/ncs/ncs.systemd.conf`, and user profile scripts from `/etc/profile.d`.
```

Comment:
This could be used for any env. config I suppose, maybe something like patching python path or similar? So, maybe better to use just:
```suggestion
Executing the above command removes the Installation Directory `/opt/ncs` including symbolic links, Configuration Directory `/etc/ncs`, Run Directory `/var/opt/ncs`, Log Directory `/var/log/ncs`, `systemd` service file `/etc/systemd/system/ncs.service`, `systemd` environment file `/etc/ncs/ncs.systemd.conf`, and user profile scripts from `/etc/profile.d`.
```